test/request_queues.test.js

```diff
@@ -492,5 +463,35 @@ describe('Request Queue methods', () => {
             expect(browserRes.items).toEqual(browserRequests);
             validateRequest({ limit: maxPageLimit }, { queueId });
         });
+
+        // NOTE: This test needs to be last otherwise it will break other tests
```

Comment:
it would be much better to remove the "dependency", tests should be ideally isolated. isn't it enough to use a different queue ID?

Reply:
The problem is how the mock server is written
Checking request from mock server is done very basic using pop() method from list, see
https://github.com/apify/apify-client-js/blob/master/test/mock_server/server.js#L48
so changing ID does not help.